backend/vecsim_app/main.py

```diff
@@ -42,34 +28,52 @@ async def startup():
     # You can set the Redis OM URL using the REDIS_OM_URL environment
     # variable, or by manually creating the connection using your model's
     # Meta object.
-    Paper.Meta.database = get_redis_connection(url=config.REDIS_URL, decode_responses=True)
+    Paper.Meta.database = get_redis_connection(
+        url=config.REDIS_URL, decode_responses=True
+    )
     await Migrator().run()
 
+
 # static image files
 app.mount("/data", StaticFiles(directory="data"), name="data")
 
 ## mount the built GUI react files into the static dir to be served.
 current_file = Path(__file__)
 project_root = current_file.parent.resolve()
 gui_build_dir = project_root / "templates" / "build"
-app.mount(
-    path="/", app=SinglePageApplication(directory=gui_build_dir), name="SPA"
-)
+app.mount(path="/", app=SinglePageApplication(directory=gui_build_dir), name="SPA")
 
 if __name__ == "__main__":
+    import logging
     import os
+
+    logging.basicConfig(level=logging.INFO)
+
     env = os.environ.get("DEPLOYMENT", "prod")
+    logging.info(f"Running in {env} mode")
 
     server_attr = {
         "host": "0.0.0.0",
         "reload": True,
-        "port": 8888,
-        "workers": 1
+        "port": int(config.SERVER_PORT),
```

Comment:
usually having problems because 8888 is typically reserved for jupyter. thus, need to expose port